src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -85,7 +91,18 @@ class SideNav extends Component {
     }
 
     return (
-      <ul className="usa-width-one-fourth va-sidenav">
+      <ul
+        className={`usa-width-one-fourth va-sidenav vads-u-margin--1 medium-screen:vads-u-height--auto medium-screen:vads-u-margin-y--0 medium-screen:vads-u-margin-left--0 medium-screen:vads-u-margin-right--2p5 ${
```

Comment:
NIT: I think this would be more readable if we extracted the base `className` into a variable. 

For example:
```
const baseClassName = `usa-width-one-fourth va-sidenav vads-u-margin--1 medium-screen:vads-u-height--auto medium-screen:vads-u-margin-y--0 medium-screen:vads-u-margin-left--0 medium-screen:vads-u-margin-right--2p5`
return (
  <ul className={baseClassName, this.state.active ? `va-sidenav-height` : null}>
    ...
)
```